guides/project-process/scrum-process.md

```diff
@@ -0,0 +1,65 @@
+# SCRUM Process
+
+Scrum is the most common Agile delivery process. For this process we need people to be Scrum Masters.
+Scrum Masters ensure the ceremonies happen in the right way at the right time.
+It is important to note that the full Scrum Process is not needed for all our project (eg IC&A).
+
+## Ceremonies
+
+The Scrum Process comprises the below ceremonies, which are described in more detail below.
+
+* Standups
+* Planning
+* Reviews/Demos
+* Retros
+* Backlog Grooming
+
+### Standups
+
+* We’re used to these
```

Comment:
We should say when these should happen
Standup once a day
Planning at the beginning of the sprint
Reviews/Demo and Retro at the end of sprint
Backlog grooming once a sprint at least